### PR TITLE
TAJO-1290: Add HBase Storage Integration Documentation

### PR DESCRIPTION
**Pull request by blrunner**
Tajo already supports HBase stroage integration. But, we don't have any documentation for the feature. Thus, we need to make a documentation for the feature.

### CODE REVIEW

**Comment by jihoonson** [comment]
Sorry for the late review.
I'm reviewing.

File: tajo-docs/src/main/sphinx/hbase_integration.rst [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+Apache Tajo™ storage supports integration with Apache HBase™.
+This integration allows Tajo to access all tables used in Apache HBase.
+
+In order to use this feature, you need to build add some configs into ``conf/tajo-env.sh`` and then add some properties into a table create statement.
+
+This section describes how to setup HBase integration.
+
+First, you need to set your HBase home directory to the environment variable ``HBASE_HOME`` in conf/tajo-env.sh as follows: ::
+
+  export HBASE_HOME=/path/to/your/hive/directory
```
Comment (jihoonson) [inline tajo-docs/src/main/sphinx/hbase_integration.rst]: ```hive``` must be changed to ```hbase```.

File: tajo-docs/src/main/sphinx/hbase_integration.rst [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+ DROP TABLE
+========================
+
+*Synopsis*
+
+.. code-block:: sql
+
+  DROP TABLE [IF EXISTS] <table_name> [PURGE]
+
+``IF EXISTS`` allows ``DROP TABLE`` statement to avoid an error which occurs when the database does not exist. ``DROP TABLE`` statement removes a table from Tajo catalog, but it does not remove the contents on HBase cluster. If ``PURGE`` option is given, ``DROP TABLE`` statement will eliminate the entry in the catalog as well as the contents on HBase cluster.
```
Comment (jihoonson) [inline tajo-docs/src/main/sphinx/hbase_integration.rst]: In the first statement, you may mean the ```table``` instead of the ```database```.

**Comment by jihoonson** [comment]
+1.
LGTM!

**Comment by blrunner** [comment]
Thanks @jihoonson 

I've just committed the patch to the master branch. :)

**Pull request closed without merging**